recipes/backports.unittest_mock/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.3" %}
+{% set sha256 = "c6d964d75112051576daa60746cc6a66afcdc43f13ea2bda175ca8e278e9b375" %}
+
+package:
+  name: backports.unittest_mock
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/b/backports.unittest_mock/backports.unittest_mock-{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools_scm >=1.15.0
```

Comment:
Please also add `backports` to the `run` *as well as* `host` requirements.
With the `backports` package we get the `backports/__init__.py` file and don't have to include this here in this package. It's important to add it to `host` as well since then `conda-build` will refrain from packaging this file. The whole deal with that is that we prevent clobbering of files from different packages, i.e., all those different `backports.*` packages providing/overriding each other's `backports/__init__.py`.